src/shogun/mathematics/linalg/ratapprox/logdet/opfunc/DenseMatrixExactLog.cpp

```diff
@@ -24,21 +24,19 @@ namespace shogun
 {
 
 	CDenseMatrixExactLog::CDenseMatrixExactLog()
-	    : COperatorFunction<float64_t>(NULL, OF_LOG)
+	    : COperatorFunction<float64_t>(nullptr, OF_LOG)
 	{
-		SG_GCDEBUG("%s created (%p)\n", this->get_name(), this)
-}
+	}
 
-CDenseMatrixExactLog::CDenseMatrixExactLog(CDenseMatrixOperator<float64_t>* op)
-	: COperatorFunction<float64_t>((CLinearOperator<float64_t>*)op, OF_LOG)
-{
-	SG_GCDEBUG("%s created (%p)\n", this->get_name(), this)
-}
+	CDenseMatrixExactLog::CDenseMatrixExactLog(
+	    CDenseMatrixOperator<float64_t>* op)
+	    : COperatorFunction<float64_t>((CLinearOperator<float64_t>*)op, OF_LOG)
+	{
```

Comment:
why the removal of the DEBUG-ing message?

Reply:
That change was requested because it was not very informative